CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to the RGSoC Teams App
+
+* Introduction, thanks for contributing
+* [Code of Conduct](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/CODE_OF_CONDUCT.md)
+
+## Reporting a bug
+
+* Very valuable
+* Check if issue already exists
+* Provide as many details as possible (screenshot)
+
+## Suggesting a new feature
+
+### Labels
+
+* Dibs
+* Ping via comment and/or join Slack
+
+## Resolving an existing issue
+
+### Finding an issue
+
+* See [`beginner-friendly`](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner-friendly) issues for your first steps in open source
+* See [`#pairwithme`](https://github.com/rails-girls-summer-of-code/rgsoc-teams/labels/%23pairwithme) for more involving issues
+* Make sure no one else is already working on it (assignee, comments)
+* Check if issue is still valid (comments)
+
+## Creating a Pull Request
+(inspired by [jbuilder](https://github.com/rails/jbuilder/blob/master/CONTRIBUTING.md))
+
+
+* If related to an issue: better ask first
+* Fork repo
+* Install locally (see [README](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/README.md) for hints)
+* Make sure tests pass locally: `rake spec`
+
+### Create a Topic Branch
+
+Make sure your fork is up-to-date and create a topic branch for your feature or bug fix.
+
+```
+git checkout master
+git pull upstream master
+git checkout -b my-feature-branch
+```
+
+### Write Code
+
+* [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide)
+* With tests (TODO: can be hard for beginners)
```

Comment:
Agreeing all over the place @carpodaster !

Maybe put it like this:
> Write tests for all your code. A well defined test suite is as much part of the app as the tested code itself. It helps to ascertain the well-being of any software product - the Teams App being no exception here. But this should not be a blocker. See the existing tests to find out where yours would fit in and in case it's hard to get started, ask us, we'll help.

Providing good but still **generic** enough test templates is hard af and I couldn't do it less than 500 loc I guess. **Guidelines** such as *"test instance methods with all possible inputs"* or *"test the response codes of all possible controller scenarios"* are probably easier to put together.